### ISSUE TITLE
Updating the website

### ISSUE THREAD

**Issue by fe51** [pyronear/pyronear.github.io#20]
In overall, our site is not up to date. Before considering a global redesign of the site, it would be nice to have the following features :

- [ ] Updates the presentation of the project 

- [ ] Adds a section listing articles where the project is mentioned, events in which Pyronear has participated

  non-exhaustive list (discuss on communication channel to be up to date):

  - [Poster](https://assets.pytorch.org/pted2021/posters/B8.png) lors de la Participation au [PyTorch Ecosystem Day](https://pytorch.org/ecosystem/pted/2021)
  - [Présentation](https://www.youtube.com/watch?v=QC52bYfXMkM&t=29931s) de la solution au dataXdays, Organisé par les équipes de Publicis Sapient
  - [Enregistrement](http://enregistrement/) d’un podcast avec horizonIA
  - [Article](https://blog.qarnot.com/les-data-scientists-sauveront-ils-le-monde/) pour le blog de Qarnot
  - [Intervention](https://www.youtube.com/watch?v=kSwJopxsphQ)au MonthlyMarty de DataForGood
  - [Intervention](https://www.youtube.com/watchlist=PLx4I2Pvlf1NIUV4dQcPolj326bf80YvrT&v=XDUVsrxHf_w&feature=youtu.be) au Ministère de la Transition Écologique, lors de la 1ère journée de la donnée au service de la transition écologique
  - Others: Epe TF1/ le parisien / hebdo Ardèche / podcastdatagen / AEFinfo

- [ ] Introduces section related "nos partenaires" (SDIS 07, etc)

- [ ] Updates consequently section "ils nous soutiennent " (D4G, Atraksis, Fondation M33)

- [ ] Adds link for donation

- [ ] Presentation of devices / Photos

- [ ] Team photo vs member photos ?

Happy to discuss it ! 😃

**Comment by frgfm** [comment]
I agree with most of what you said :) 
Only part I have doubts is "team photo vs. member photos":
- going for member photos can end up in a "badge" for some people and not others. So I'd argue against
- now for team, the same applies but especially it "exposes" people (for people looking for customer support let's say). Additionally, I do think that the project should feel like going beyond the core contributors (e.g. also the people opening a PR only once on GitHub, etc.).

What do you think?

**Comment by frgfm** [comment]
Sorry about the late reply, I agree, let's do a general team picture if people on it are alright with that :smiley: 

If we agree on the other modifications, I can start opening PRs in the coming days :+1:
